apis/run/v1alpha2/tanzukubernetesrelease_types.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Copied this over from TKG Service. Practically identical to the v1alpha1 version.